model/utils.go

```diff
@@ -65,9 +67,23 @@ func NewAppError(where string, message string, details string) *AppError {
 	ap.Where = where
 	ap.DetailedError = details
 	ap.StatusCode = 500
+	ap.IsOAuth = false
 	return ap
 }
 
+func Md5Encrypt(salt string, text string) string {
```

Comment:
MD5 is a hash, which explains why there is no decrypt. But it doesn't explain why this function is called encrypt.